diff -Naur a/Makefile b/Makefile
--- a/Makefile	2014-12-05 15:32:21.000000000 +0200
+++ b/Makefile	2014-12-06 02:01:30.679893494 +0200
@@ -42,7 +42,7 @@
 VDRCONFDIR = $(call PKGCFG,configdir)
 PLGRESDIR = $(call PKGCFG,resdir)/plugins/$(PLUGIN)
 TMPDIR ?= /tmp
-SKINFLATPLUS_WIDGETDIR ?= $(LIBDIR)/$(PLUGIN)/widgets
+SKINFLATPLUS_WIDGETDIR = /etc/vdr/plugins/skinflatplus/widgets
 
 ### The compiler options:
 export CFLAGS   = $(call PKGCFG,cflags)
