diff -Naur a/drivers/media/pci/saa716x/saa716x_ff_ir.c b/drivers/media/pci/saa716x/saa716x_ff_ir.c
--- a/drivers/media/pci/saa716x/saa716x_ff_ir.c	2018-02-06 00:25:20.000000000 +0200
+++ b/drivers/media/pci/saa716x/saa716x_ff_ir.c	2018-02-08 00:41:49.934817282 +0200
@@ -22,7 +22,6 @@
 
 #include <linux/types.h>
 #include <linux/input.h>
-
 #include "saa716x_spi.h"
 #include "saa716x_priv.h"
 #include "saa716x_ff.h"
@@ -203,9 +202,7 @@
 	if (!ir)
 		return -ENOMEM;
 
-	init_timer(&ir->keyup_timer);
-	ir->keyup_timer.function = ir_emit_keyup;
-	ir->keyup_timer.data = (unsigned long) ir;
+	//timer_setup(&ir->keyup_timer, void *ir_emit_keyup, 0);
 
 	input_dev = input_allocate_device();
 	if (!input_dev)
@@ -237,8 +234,7 @@
 	ir_register_keys(ir);
 
 	/* override repeat timer */
-	input_dev->timer.function = ir_repeat_key;
-	input_dev->timer.data = (unsigned long) ir;
+	//timer_setup(&input_dev->timer, void *ir_repeat_key, 0);
 
 	tasklet_init(&ir->tasklet, ir_emit_key, (unsigned long) saa716x);
 	saa716x->ir_priv = ir;
